src/toolchain/toolchain.ts

```diff
@@ -35,29 +35,37 @@ export class SwiftToolchain {
         public swiftVersion: Version,
         public toolchainPath?: string,
         public developerDir?: string,
```

Comment:
Is it necessary to store the developer directory anymore? It is only needed temporarily when calculating the xctest folder for Xcode. We could move the `xcode-select -p` call inside `getXCTestPath`.

Reply:
I don't think it is - but I don't know why it was being saved in the first place.

Reply:
Originally it was stored so we could calculate the xcTestPath in a different part of the codebase. With all these collected together these is no need to store it.